packages/augur-test/src/tests/state/getter/Markets.test.ts

```diff
@@ -1,1032 +1,1839 @@
+        numShares,
+        price,
+        outcome1,
+        stringTo32ByteHex(''),
+        stringTo32ByteHex(''),
+        stringTo32ByteHex('42')
+      );
+
+      const priceAdjustment = new BigNumber(1);
+      const createOrder = (orderType: BigNumber) => (
+        price: BigNumber,
+        outcome: BigNumber
+      ) =>
+        john.placeOrder(
+          yesNoMarket.address,
+          orderType,
+          numShares,
+          price,
+          outcome,
+          stringTo32ByteHex(''),
+          stringTo32ByteHex(''),
+          stringTo32ByteHex('42')
+        );
+
+      const createAsks = createOrder(ORDER_TYPES.ASK);
+      const createBids = createOrder(ORDER_TYPES.BID);
+
+      await Promise.all([
+        // Outcome 0
+        // Price level 1
+        createAsks(price, outcome0),
+        createAsks(price, outcome0),
+        createAsks(price, outcome0),
+
+        createBids(price, outcome0),
+        createBids(price, outcome0),
+        createBids(price, outcome0),
+
+        // Price level 2
+        createAsks(price.plus(priceAdjustment), outcome0),
+        createAsks(price.plus(priceAdjustment), outcome0),
+
+        createBids(price.minus(priceAdjustment), outcome0),
+        createBids(price.minus(priceAdjustment), outcome0),
+
+        // Outcome 1
+        // Price level 1
+        createAsks(price, outcome1),
+        createAsks(price, outcome1),
+        createAsks(price, outcome1),
+
+        createBids(price, outcome1),
+        createBids(price, outcome1),
+        createBids(price, outcome1),
+
+        // Price level 2
+        createAsks(price.plus(priceAdjustment), outcome1),
+        createAsks(price.plus(priceAdjustment), outcome1),
+
+        createBids(price.minus(priceAdjustment), outcome1),
+        createBids(price.minus(priceAdjustment), outcome1),
+      ]);
+
+      await (await db).sync(john.augur, mock.constants.chunkSize, 0);
+    }, 120000);
+
+    test('should require marketId', async () => {
+      await expect(api.route('getMarketOrderBook', {})).rejects.toThrowError();
+    });
 
-  expect(markets[0].reportingState).toBe(MarketInfoReportingState.PRE_REPORTING);
-  expect(markets[1].reportingState).toBe(MarketInfoReportingState.PRE_REPORTING);
-  expect(markets[2].reportingState).toBe(MarketInfoReportingState.PRE_REPORTING);
+    describe('outcomeId', () => {
+      test('can be a single value', async () => {
+        const orderBook = (await api.route('getMarketOrderBook', {
+          marketId: yesNoMarket.address,
+          outcomeId: outcome0.toNumber(),
+        })) as MarketOrderBook;
+
+        expect(orderBook.orderBook).toEqual(
+          expect.objectContaining({
+            0: expect.objectContaining({}),
+          })
+        );
+      });
+
+      test('can be an array', async () => {
+        const orderBook = (await api.route('getMarketOrderBook', {
+          marketId: yesNoMarket.address,
+          outcomeId: [outcome0.toNumber(), outcome1.toNumber()],
```

Comment:
Nit: It might be more intuitive to just call this `outcomeIds` and have it either be an array or be omitted.

Reply:
Fair point. I *think reportingState follows the same pattern. So I just followed along.